aybapi/impl/client.py

```diff
@@ -0,0 +1,47 @@
+import typing
+
+# Importing implemented models
+from .bot import Bot
+from .category import Category
+
+# Importing helper methods and utils
+from ..utils.endpoints import *
+from ..utils.fetch import fetch
+
+# Importing base class of Client model
+from ..base.client import Client as ClientBase
+
+
+class Client(ClientBase):
+    def fetch_stats(self) -> typing.Optional[dict]:
```

Comment:
`dict` is the same as `typing.Dict[typing.Any, typing.Any]`, which is not the correct typing for what you want.

Also the docstring says you return a dict, the typehint says you return a dict or None.